lib/helmsman_web/templates/page/index.html.eex

```diff
@@ -4,6 +4,7 @@
   <ul>
     <%= for cluster <- @clusters do %>
       <li><%= cluster.name %> - <%= connection_strings(cluster.configured_brokers) %></li>
+      <li><%= cluster.name %> - <%= connection_strings(cluster.advertised_brokers) %></li>
```

Comment:
This is gonna look a little weird with each cluster listed twice.  It should maybe change to a dl or something?

Reply:
I totally agree, I'll clean it up.